test/unit/manipulation.js

```diff
@@ -2162,6 +2162,16 @@ testIframeWithCallback(
 	}
 );
 
+testIframeWithCallback(
+	"domManip executes scripts in iframes in the iframes' context",
+	"manipulation/scripts-context.html",
+	function( bodyElement, html, assert ) {
+		assert.expect( 1 );
+		jQuery( bodyElement ).append( html );
+		assert.ok( !window.scriptTest, "script executed in iframe context" );
```

Comment:
Good call. Added the iframe window to arguments to make sure the variable is on that window.